core/src/main/java/com/novoda/accessibility/ActionsAlertDialogCreator.java

```diff
@@ -14,29 +16,37 @@
 
     private final Actions actions;
 
-    public ActionsAlertDialogCreator(Context context, int title, Actions actions) {
+    public ActionsAlertDialogCreator(Context context, Actions actions) {
+        this(context, NO_TITLE, actions);
+    }
+
+    public ActionsAlertDialogCreator(Context context, @StringRes int title, Actions actions) {
         this.context = context;
         this.title = title;
         this.actions = actions;
     }
 
     public AlertDialog create() {
-        return new AlertDialog.Builder(context)
-                .setTitle(title)
-                .setItems(
-                        collateActionLabels(),
-                        new DialogInterface.OnClickListener() {
-
-                            @Override
-                            public void onClick(DialogInterface dialog, int which) {
-                                Action action = actions.getAction(which);
-                                action.run();
-                                dialog.dismiss();
-                            }
-
-                        }
-                )
-                .create();
+        AlertDialog.Builder builder = new AlertDialog.Builder(context);
+        builder.setItems(
+                collateActionLabels(),
+                new DialogInterface.OnClickListener() {
+
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        Action action = actions.getAction(which);
+                        action.run();
+                        dialog.dismiss();
+                    }
+
+                }
+        );
+
+        if (title != 0) {
```

Comment:
Fixed in https://github.com/novoda/accessibilitools/pull/7/commits/c94e62a57788fd006b452e581a840d27dd6fb98c